frontend/institution/manageInstMenu/manageInstMenuController.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
+
+(function() {
+    angular
+    .module("app")
+    .controller("ManageInstMenuController", [
+        'AuthService', 'ManageInstItemsFactory',
+        ManageInstMenuController]);
+
+    function ManageInstMenuController(AuthService, ManageInstItemsFactory) {
+        const manageInstMenuCtrl = this;
+
+        manageInstMenuCtrl.$onInit = () => {
+            _.defaults(manageInstMenuCtrl, {
+                user: AuthService.getCurrentUser(),
+            });
+            
+            manageInstMenuCtrl._loadSwitchInstOptions();
+            manageInstMenuCtrl._loadInstitution();
+        };
+
+        manageInstMenuCtrl._loadInstitution = () => {
+            const currentProfile = manageInstMenuCtrl._getProfilesAdmin()
+                .find(prof => prof.institution_key === manageInstMenuCtrl.user.current_institution.key);
+                
+            manageInstMenuCtrl.institution = currentProfile.institution;
+            manageInstMenuCtrl._loadMenuOptions();
+        };
+
+        manageInstMenuCtrl._loadMenuOptions = () => {
+            // the slice was used to get just the first four items
```

Comment:
and why just the first 4 items?

Reply:
Because the last item is the return button and it is not necessary on this menu

Reply:
put this in the comment please